packages/core/src/fhir-to-cda/README.md

```diff
@@ -0,0 +1,55 @@
+Miminimum Required Composition
+
+This is a combination of required from the FHIR standard perspective and also the Metriport Business
```

Comment:
I see how the FHIR requirements can determine what's minimum on the Composition, but as a developer it would be useful to know what's the minimum requirement for CDA. This is dependent on each resource, right?

Can we add info here about where to look for the spec and identify what's needed on each resource/property?

For example: I see we're using `withoutNullFlavorString` to build the `state` prop of address [here](https://github.com/metriport/metriport/pull/1757/files#diff-d3cd287f58f63f13ff78e0f4ad17acbcc890ec44e4f4a84666826ef27e0e26a9R162). This is a flag for me, as I find it odd that the state (and other properties) can be defined either with a string or a Map interchangeably. I'd like to check on the spec to see if that's expected or not.

Reply:
So this is a good example to see what is expected: 

```
<addr use="HP">
    <streetAddressLine>1234 Test Lane</streetAddressLine>
    <city>SACRAMENTO</city>
    <state>CA</state>
    <postalCode>95836</postalCode>
    <useablePeriod xsi:type="IVL_TS"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
        <low value="20090819" />
        <high value="20220525" />
    </useablePeriod>
</addr>
```

Some fields just want text inside the tag (`<streetAddressLine>`) and some want an attribute - like value - inside (`<low>`)
You can really dive into this in depth in places like this, https://build.fhir.org/ig/HL7/CDA-core-sd/StructureDefinition-AD.html, but it gets exceedingly complicated and the approach I think we are taking is moreso just emulating what we see Epic Generating in its XMLs + cross referencing the spec, as opposed to going through the spec line by line and trying to make everything exactly conformant, since its a pretty deep rabbit hole